Mysql Inventory Bridge/src/main/resources/plugin.yml

```diff
@@ -4,11 +4,12 @@ version: "{version}"
 authors:
   - CraftersLand
   - RicardormDev
+  - Hugo1307
 website: www.craftersland.net & mymetaverse.io
 load: STARTUP
 api-version: 1.16
 description: Sync players inventorys over many servers using MySQL. This is an edited version for and by MyMetaverse
-softdepend: [ProtocolLib]
+softdepend: [ProtocolLib, MetaWallet]
```

Comment:
If MetaWallet is softdepend let's add a checker before creating the instance and using the instance, otherwise the plugin will run lots of errors.
It's a good idea that this dependency is softdepend that way we can scale it to any type of server.

Reply:
Huum, it was already added as a soft depend. Also, I'm verifying on WalletHandler if the MetaWallet is enabled. If it doesn't, the plugin won't try to use it as a dependency.